vortex-array/src/compute/binary_numeric.rs

```diff
@@ -138,7 +142,7 @@ pub fn binary_numeric(
 
     // Check if RHS supports the operation directly.
     if let Some(fun) = rhs.encoding().binary_numeric_fn() {
-        if let Some(result) = fun.binary_numeric(rhs, lhs, op)? {
+        if let Some(result) = fun.binary_numeric(rhs, lhs, op.flip_parameters())? {
```

Comment:
Can you be consistent with naming for CompareFn, in this case `op.swap()`

Reply:
Done.